test/system/300-cli-parsing.bats

```diff
@@ -158,6 +158,18 @@ EOF
     expect[weird*na#me!]=$weirdname
 
     _check_env $resultsfile
+
+    # Now check the same with podman exec
+    run_podman run -d --name testctr        \
+            -v "$resultsfile:/envresults:Z" \
+            $IMAGE top
+
```

Comment:
Is `wait_for_output "Load average:"` needed here, to guarantee that the volume is mounted and the container is ready for an `exec`? (I think not, i.e. I think what you're doing is safe, but would like explicit confirmation).

Reply:
It should not be needed here, podman run -d waits for the process to be started before returning which means all mounts etc are setup already. It is only needed the test depends on the container process to do something.